CODEOWNERS

```diff
@@ -1,4 +1,74 @@
 # Users referenced in this file will automatically be requested as reviewers for PRs that modify the given paths.
 # See https://help.github.com/articles/about-code-owners/
 
-* @dotnet/dotnet-docker-reviewers
+### General infra ###
+/* @dotnet/dotnet-docker-reviewers
+eng/ @dotnet/dotnet-docker-reviewers
+
+### Dockerfiles ###
+
+# almalinux
+src/almalinux/**/source-build/ @dotnet/source-build-internal
+src/almalinux/**/helix/ @sbomer
+
+# alpine
+src/alpine/**/helix/ @dotnet/runtime-infrastructure @dotnet/ncl
+src/alpine/**/amd64/ @dotnet/dotnet-source-build-internal

```

Comment:
https://github.com/dotnet/dotnet-buildtools-prereqs-docker/issues/1148 discusses the use cases for these owners.  The current two use cases involves assigning issues to these owners.  To my knowledge, GH issues cannot be assigned to a team.  This is mentioned as a consideration in the issue.  What are your thoughts on how to handle this?

Reply:
I realized this. `dotnet/dnceng` uses the policy of pinging the team on their issues, so I thought we could do something similar if a team is assigned and not an individual. Also, assigning individuals can get tricky if someone is OOF or non-responsive. In that instance, we'd have to track down the team they belong to and ping them anyways.

Reply:
Can you capture this in the issue?  I agree with all the downsides of assigning these types of issues to an individual.

Reply:
This begs the question if the policy should prevent individuals and only allow teams?

Reply:
> This begs the question if the policy should prevent individuals and only allow teams?

I don't see why not. That said, I'm not entirely sure how we should decipher between an individual and a team... I suppose teams have the `@dotnet/` text at the beginning of the name so maybe we check if it's a team based on that?